mscore/data/shortcuts.xml

```diff
@@ -982,6 +982,14 @@
     <key>toggle-insert-mode</key>
     <seq>Ctrl+I</seq>
     </SC>
+  <SC>
```

Comment:
Given the different keyboard layouts—as discussed at https://musescore.org/en/node/152766—I know it's not as "sexy" to add a feature that the user has to enable, but it seems smartest to leave it to the users to define their own shortcuts. Thank you for your idea and coding!